LoopKit/QuantityFormatter.swift

```diff
@@ -156,6 +156,15 @@ private extension HKUnit {
             }
         }
 
+        if self == .internationalUnitsPerHour {
+            switch style {
+            case .short, .medium:
+                return LocalizedString("U/hr", comment: "The short unit display string for international units of insulin per hour")
+            case .long:
+                return LocalizedString("Units/Hour", comment: "The long unit display string for international units of insulin per hour")
```

Comment:
I don't think "Hour" should be capitalized.

Do we need to have a pluralized version? So, "1 Unit/hour" and "2 Units/hour"?